qgreenland/util/config/export.py

```diff
@@ -82,7 +82,9 @@ def export_config_csv(
         if not layer_cfg.in_package:
             continue
 
-        layer_type: QgsLayerType
+        # NOTE: we want to make it clear to users that some layers are "online"
+        # and we use the "Vector or Raster" column to do that.
+        layer_type: Union[QgsLayerType, Literal["Online"]]
```

Comment:
I'm not sure how I feel about moving `"Online"` out of `QgsLayerType`. The name corresponds with the field `layer_type`, so it seems a bit odd to extend it to make it work with the field. Maybe we should rename `QgsLayerType` to something more like `VectorOrRaster` that makes it more clear what its purpose is?

Reply:
Renaming from `QgsLayerType` -> `VectorOrRaster` would certainly be more clear! I think making that change makes sense. 

There's also a concept of `LayerType` in PyQGIS that might get conflated with our custom type: https://qgis.org/pyqgis/master/gui/Qgis.html#qgis.gui.Qgis.LayerType